pulumi/folio/folio.ts

```diff
@@ -169,14 +175,23 @@ export module deploy {
 
             fullnameOverride: module.name,
 
-            // TODO Turn this back on after figuring out how to secure it. It is known to work at port 80.
-            // For how to enable SSL see https://aws.amazon.com/premiumsupport/knowledge-center/terminate-https-traffic-eks-acm/.
-            // service: {
-            //     type: "LoadBalancer",
-            //     // TODO Will probably want to make this 443.
-            //     port: 9130,
-            //     containerPort: 9130
-            // },
+            // For documentation on the annotations and other configuration options see:
+            // https://aws.amazon.com/premiumsupport/knowledge-center/terminate-https-traffic-eks-acm/
+            service: {
+                type: "LoadBalancer",
+                // This port must be 9130 since that is where the other pods in the cluster
+                // expect OKAPI to be available. This means that all external requests to okapi
+                // must have the port number on the URL.
+                port: 9130,
+                containerPort: 9130, // Maps to targetPort in the spec.ports array in folio-helm.
+                annotations: {
+                    "service.beta.kubernetes.io/aws-load-balancer-backend-protocol": "http",
+                    "service.beta.kubernetes.io/aws-load-balancer-ssl-cert": `${certArn}`,
+                    // Only run ssl on port named http. This is the only port name that folio-helm
+                    // makes available.
+                    "service.beta.kubernetes.io/aws-load-balancer-ssl-ports": "http"
+                }
```

Comment:
Do we want OKAPI publicly available? Is that needed for API access?

Reply:
Yes we need this port to be open to get this simple setup. I don't think it is any less secure than having for example an ingress controller at 443 map to 9130 inside the cluster. The port here can't be 443 because other pods want to talk to okapi over 9130. That's what the comment is trying to get across. I think we have the security benefits of the load balancer either way.